src/control/cmd/dmg/pretty/system.go

```diff
@@ -222,24 +222,16 @@ func PrintSystemCleanupResponse(out io.Writer, resp *control.SystemCleanupResp,
 	fmt.Fprintln(out, "System Cleanup Success")
 }
 
-// PrintSystemDrainResponse generates a human-readable representation of the supplied
-// SystemDrainResp struct and writes it to the supplied io.Writer. Result related errors written to
-// error io.Writer.
-func PrintSystemDrainResponse(out io.Writer, resp *control.SystemDrainResp) {
-	if len(resp.Results) == 0 {
-		fmt.Fprintln(out, "No pool ranks drained")
-		return
-	}
-
+func printSysOsaResults(out io.Writer, results []*control.SystemOsaResult) {
```

Comment:
This helper should have a comment. Also, what is `Osa`? Maybe a better name for that whole concept and family of types/functions might be `PoolMembershipUpdates`? Ultimately, the functionality is less about the system and more about the changes to pools, right?